pages/platform/ai/cli_18_howto_deploy_app/guide.en-gb.md

```diff
@@ -0,0 +1,283 @@
+  -c, --cpu <CPU>                                                                                               Number of CPUs (ignored if GPUs is specified)
+  -v, --volume <container@alias/prefix:mount_path(:permission)(:cache) or url:mount_path(:permission)(:cache)>  Volumes mounted on the image. `alias` is the data store alias of your data. You can get a list of all available data stores for the object storage by typing `ovhai data store list`. `/prefix` is optional [default: ""]. `:permission` is optional [default: ro] [possible values: ro, rw, rwd]. `:cache` is optional [default: "no-cache"] [possible values: cache, no-cache]
+  -n, --name <NAME>                                                                                             Optional name, only informative
+  -l, --label <name=value>                                                                                      Optional labels, only informative
+  -o, --output <OUTPUT>                                                                                         Command output format [possible values: json, yaml]
+      --replicas <REPLICAS>                                                                                     Fixed scaling strategy: Number of replicas (defaults to 1)
+      --auto-min-replicas <MIN REPLICAS>                                                                        Automatic scaling strategy: Minimum number of replicas [aliases: min-replicas]
+      --auto-max-replicas <MAX REPLICAS>                                                                        Automatic scaling strategy: Maximum number of replicas [aliases: max-replicas]
+      --auto-resource-type <TYPE>                                                                               Automatic scaling strategy: Resource type to base the automatic scaling on [aliases: resource-type] [possible values: CPU, RAM]
+      --auto-resource-usage-threshold <THRESHOLD>                                                               Automatic scaling strategy: Average resource usage threshold (in percents) at which the number of replicas will be automatically increased [aliases: resource-threshold]
+      --probe-path <PATH>                                                                                       Path to access (defaults to /)
+      --probe-port <PORT>                                                                                       Number of the port to access (defaults to --default-http-port)
+      --no-color                                                                                                Remove colors from output
+  -h, --help                                                                                                    Print help (see more with '--help')
+```
+
+The `IMAGE` argument is mandatory. Indeed, you need to specify a Docker image that you either built yourself or find freely available on a public repository such as [DockerHub](https://hub.docker.com/). 
```

Comment:
I might have added a "primary flag" (in blue) just below, redirecting you to your registries doc.